hawk_eye/data_generation/slice_image.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+""" Contains code to slice up an image into smaller tiles. """
+
+import argparse
+import pathlib
+from typing import List, Tuple
+
+import numpy as np
+from PIL import Image
+from tqdm import tqdm
```

Comment:
Just import modules and packages, not classes or functions.

Reply:
Looks like you have to import PIL.Image as `from PIL import Image`. Is that right?